### PR TITLE
Notify test automation engineers about changes in the repository

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
As agreed with the test automation engineers, this pull request updates the `CODEOWNERS` file to include their gitHub team so the are notified about changes in this repository.

### CODE REVIEW

**Comment by CarlosCortizasCT** [comment]
@tdeekens @krishhna123 This PR was a follow-up on the conversation we had around test-automation engineers being more involved with the changes in this repository.

It's been a while waiting for approvals or feedback. Is there anything you think should be reconsidered?

**Review by tdeekens — APPROVED** [review]

**Review by krishhna123 — APPROVED** [review]

**Pull request merged**
